server/internal/app/repo.go

```diff
@@ -35,6 +39,27 @@ func initReposAndGateways(ctx context.Context, conf *Config) (*repo.Container, *
 		log.Fatalf("Failed to init mongo: %+v\n", err)
 	}
 
+	// File
+	datafs := afero.NewBasePathFs(afero.NewOsFs(), "data")
+	var fileRepo gateway.File
+	if conf.GCS.BucketName == "" {
+		log.Infoln("file: local storage is used")
+		fileRepo, err = fs.NewFile(datafs, conf.AssetBaseURL)
+	} else {
+		log.Infof("file: GCS storage is used: %s\n", conf.GCS.BucketName)
+		fileRepo, err = gcs.NewFile(conf.GCS.BucketName, conf.AssetBaseURL, conf.GCS.PublicationCacheControl)
+		if err != nil {
+			if debug {
+				log.Warnf("file: failed to init GCS storage: %s\n", err.Error())
+				err = nil
+			}
+		}
+	}
```

Comment:
```suggestion
	var fileRepo gateway.File
	if conf.GCS.BucketName == "" {
		log.Infoln("file: local storage is used")
		datafs := afero.NewBasePathFs(afero.NewOsFs(), "data")
		fileRepo, err = fs.NewFile(datafs, conf.AssetBaseURL)
	} else {
		log.Infof("file: GCS storage is used: %s\n", conf.GCS.BucketName)
		fileRepo, err = gcs.NewFile(conf.GCS.BucketName, conf.AssetBaseURL, conf.GCS.PublicationCacheControl)
		if err != nil {
			log.Fatalf("file: failed to init GCS storage: %s\n", err.Error())
		}
	}
```

- datafs is not used when GCS is used. so it should move inside the more narrow scope
- panic when GCS is failed to be initialized

Reply:
Done!